### PR TITLE
Twenty news demo

### PR DESCRIPTION
**Pull request by hectorgon**
Work in progress, Chris if I don't finish it today you can take over

@christhetree @deerzq

### CODE REVIEW

**Comment by hectorgon** [comment]
@deerzq this PR is ready for review

File: demo/twenty_news/src/main/scala/TwentyNewsPipeline.scala [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+
+object TwentyNewsPipeline {
+  val log: Logger = LoggerFactory.getLogger(this.getClass)
+
+  def debugExampleRun(sc : SparkContext, config : Config) = {
+    val inputFile : String = config.getString("input")
+    val count : Int = config.getInt("count")
+    val example = sc.textFile(inputFile)
+      .map(lineToExample)
+      .filter(x => x !=	None)
```
Comment (deerzq) [inline demo/twenty_news/src/main/scala/TwentyNewsPipeline.scala]: space: `x != None`

File: demo/twenty_news/src/main/scala/TwentyNewsPipeline.scala [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+  }
+
+  def lineToExample(line: String) : Option[Example] = {
+    val example = new Example()
+    val featureVector = new FeatureVector()
+    example.addToExample(featureVector)
+    // set string features
+    val stringFeatures = new java.util.HashMap[String, java.util.Set[java.lang.String]]()
+    featureVector.setStringFeatures(stringFeatures)
+    val strFv = new java.util.HashSet[java.lang.String]() // education features
```
Comment (deerzq) [inline demo/twenty_news/src/main/scala/TwentyNewsPipeline.scala]: remove the comment `// eduction ...`

**Comment by deerzq** [comment]
Nice demo! Just some nits, LGTM

File: demo/twenty_news/src/main/scala/TwentyNewsPipeline.scala (hunk 1 above)
Comment (hectorgon) [inline demo/twenty_news/src/main/scala/TwentyNewsPipeline.scala]: Done

File: demo/twenty_news/src/main/scala/TwentyNewsPipeline.scala (hunk 2 above)
Comment (hectorgon) [inline demo/twenty_news/src/main/scala/TwentyNewsPipeline.scala]: Done

**Pull request merged**